src/index.html

```diff
@@ -27,8 +27,8 @@
   <script src="/assets/js/soundmeter.js" type="text/javascript"></script>
   <script type="module" src="/assets/js/emitter.js"></script>
   <script type="module" src="/assets/js/main.js"></script>
-  <script type="module" src="/assets/js/events.js"></script>
-  <!--<script type="module" src="/assets/js/rtc.js"></script> -->
+  <!-- <script type="module" src="/assets/js/events.js"></script> -->
```

Comment:
Can be removed is commented out and dead code, git has a history in case we wanna get it back later on

Reply:
- [x] patch myself